src/Security/Group.php

```diff
@@ -725,6 +726,7 @@ public function requireDefaultRecords()
      */
     private function dedupeCode(): void
     {
+        Deprecation::notice('4.12.0', 'Remove deduping in favour of throwing a validation error for duplicates.');
```

Comment:
```suggestion
        Deprecation::notice('4.12.0', 'Replaced with a validation error when a duplicate is found.');
```
Also update the version and message in the phpdoc

Reply:
Updated